apps/test/util/testUtils.js

```diff
@@ -291,3 +291,30 @@ export function throwOnConsoleWarnings() {
     firstError = null;
   });
 }
+
+/**
+ * Stub everything on window.Applab for every test in this describe block.
+ */
+export function stubWindowApplab() {
```

Comment:
This is stubbing Applab in a very specific way.

For example here https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/applab/dataCallbacksTests.js#L24 is somewhere that it gets stubbed differently.

I wonder if this method should either be renamed, or changed so that the caller passes in the stubbed value (i.e. `{ executeCmd() {} }`). Thoughts?